mlperf_logging/rcp_checker/training_1.1.0/rcps_maskrcnn.json

```diff
@@ -3,13 +3,13 @@
   "maskrcnn_ref_48":
   {
     "Benchmark": "maskrcnn",
+    "Creator": "NVidia",
+    "When": "Prior to 1.0 submission",
+    "Platform": "TBD",
     "BS": 48,
     "Hyperparams": {
-      "opt_learning_decay_factor": 0.1,
```

Comment:
Is removing `opt_learning_decay_factor`, `max_image_size` and `min_image_size` discussed in the training WG?